client/src/components/course-results-view/SisuExportDialog.tsx

```diff
@@ -159,22 +161,29 @@ const SisuExportDialog = ({ open, handleClose }) => {
                 InputLabelProps={{ shrink: true }}
                 type="date"
                 label="Assessment Date"
-                format="DD-MM-YYYY"
+                /* TODO: Fix TS */
+                //format="DD-MM-YYYY"
```

Comment:
`format` doesn't appear to be a property of `TextField` at all: https://mui.com/toolpad/reference/components/text-field/

This doesn't seem to format the date entry from what I tried even in the JavaScript version, so it seems to me that this is just an uncaught error. If so, these lines should just be removed and we can have another PR where we fix all of the date entries, there seem to be others that are also formatted as MM-DD-YYYY.